src/Nordea/Components/TextInput.elm

```diff
@@ -71,6 +73,8 @@ type alias Config msg =
     , maxLength : Maybe Int
     , pattern : Maybe String
     , hasSearchIcon : Bool
+    , hasRemoveInputButton : Bool
+    , onRemoveInput : Maybe msg
```

Comment:
Is there a need to have both fields, or can the bool be inferred internally from the other? E.g., `hasRemoveInputButton = Maybe.isJust onRemoveInput`

Reply:
I think we need both fields since we can have some condition (for example no input typed) when we don't want to show the button. But once there is some input is typed we want to show the button.

Reply:
Hmm, what if we standardized that behaviour and made it internal to this component? In other words, this component checks if the input is empty or not and shows/hides the button accordingly. Do you think that'll work?

Reply:
I'll try to implement that.

Reply:
I implemented it so that we're checking internally if onClearInput is Just and input is non-empty when showing the button.